### PR TITLE
rm: Add documenation for certificate rotation

### PR DESCRIPTION
**Pull request by doanac**
Signed-off-by: Andy Doan <andy@foundries.io>

## Readiness

* [ ] Merge (pending reviews)
* [ ] Merge after _date or event_
* [ ] Draft

## Overview

_Why merge this PR? What does it solve?_

## Checklist

_Optional. Add a 'x' to steps taken._
_You can fill this out after opening the PR. "Did I..."_

* [ ] Run spelling and grammar check, preferably with linter.
* [ ] Avoid changing any header associated with a link/reference.
* [ ] Step through instructions (or ask someone to do so).
* [ ] Review for [wordiness](https://languagetool.org/insights/post/wordiness/)
* [ ] Match tone and style of page/section.
* [ ] Run `make linkcheck`.
* [ ] View HTML in a browser to check rendering.
* [ ] Use [semantic newlines](https://bobheadxi.dev/semantic-line-breaks/).
* [ ] follow best practices for commits.
  * [ ] Descriptive title written in the imperative.
  * [ ] Include brief overview of QA steps taken.
  * [ ] Mention any related issues numbers.
  * [ ] End message with sign off/DCO line (`-s, --signoff`).
  * [ ] Sign commit with your gpg key (`-S, --gpg-sign`).
  * [ ] Squash commits if needed.
* [ ] Request PR review by a technical writer and at least one peer.

## Comments

_Any thing else that a maintainer/reviewer should know._
_This could include potential issues, rational for approach, etc._

### CODE REVIEW

**Comment by doanac** [comment]
@kprosise - i've created this PR as an alternative to https://github.com/foundriesio/docs/pull/484. This one splits up content into a user guide and reference manual. I'm not sure which is better. I like this split, but I worry its making the two individual sections look inadequate.

**Comment by kprosise** [comment]
I will take a look soon! Thanks for doing this!

**Comment by kprosise** [comment]
@doanac I don't see a separate UG page, does the current page here just need to be moved over?

**Comment by doanac** [comment]
> @doanac I don't see a separate UG page, does the current page here just need to be moved over?

sorry - fixed in force push.

**Comment by doanac** [comment]
Docs for 08beaf9b89bf43d661812dae6e0434b7b635b6b1 are browsable at: https://ci.foundries.io/projects/fio-docs/builds/1949/docs/artifacts/html/index.html

File: source/user-guide/cert-rotation.rst [hunk 1]
```diff
@@ -0,0 +1,66 @@
+.. _ref-cert-rotation-ug:
+
+Device Certificate Rotation
+===========================
+
+The :ref:`Device Certificate Rotation <ref-cert-rotation>` reference manual describes core concepts and functions of certificate rotation.
+This page explains how to configure your factory for this functionality.
```
Comment (kprosise) [inline source/user-guide/cert-rotation.rst:7]:
```suggestion
This page explains how to configure your Factory for this functionality.
```

File: source/user-guide/cert-rotation.rst [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+Choosing an EST Server
+----------------------
+
+Before you can perform certificate rotations, you must ensure you have taken control of your factory's :ref:`PKI <ref-device-gateway>`.
```
Comment (kprosise) [inline source/user-guide/cert-rotation.rst:12]:
```suggestion
Before you can perform certificate rotations, you must ensure you have taken control of your Factory's :ref:`PKI <ref-device-gateway>`.
```

File: source/user-guide/cert-rotation.rst [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+FoundriesFactory Managed
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Running ``fioctl keys est authorize`` will allow FoundriesFactory to run an EST server for you at ``<repoid>.est.foundries.io``.
+This command will sign a CSR created in the backend with your Factory's root key.
+The resulting TLS certificate will be used by the FoundriesFactory EST server.
+
+.. note::
+   This option requires the FoundriesFactory backend to have a certificate authority to sign renewal requests.
```
Comment (kprosise) [inline source/user-guide/cert-rotation.rst:24]:
```suggestion
   This option requires the FoundriesFactory® backend to have a certificate authority to sign renewal requests.
```

**Review by kprosise — APPROVED** [review]
LGTM

**Comment by doanac** [comment]
So you prefer this approach as opposed to the other one?

**Comment by kprosise** [comment]
> So you prefer this approach as opposed to the other one?

Yep. If we wanted, both pages could have a "seealso" right at top linking to the other, though I do not think that is needed. I do like having them split apart..easy enough to dive into the RM if more info is needed, while not overwhelming the reader with info, something I find is easy to do.

**Comment by doanac** [comment]
Closed in favor of https://github.com/foundriesio/docs/pull/493/

**Comment by kprosise** [comment]
@doanac I noticed this one was closed but 484 is still open. Is the plan to force these changes to the other PR? Just wanted to check in!

**Comment by doanac** [comment]
oops - closed the wrong one.

**Pull request merged**